packages/react-data-grid/src/__tests__/CellExpand.spec.js

```diff
@@ -1,68 +1,60 @@
 import React from 'react';
```

Comment:
Can we rename the test file to `CellExpander.spec.js` to match the component file